src/components/multi-action-button/multi-action-button.component.js

```diff
@@ -6,20 +6,29 @@ import StyledMultiActionButton from './multi-action-button.style';
 import Button from '../button';
 
 class MultiActionButton extends SplitButton {
+  /**
+   * Returns the props for the main button.
```

Comment:
We can think about changing multipleline comments into one line comments if it is possible